discovery-provider/src/tasks/index_user_bank.py

```diff
@@ -228,13 +280,23 @@ def process_user_bank_tx_details(
 
     elif has_transfer_instruction:
         # Accounts to refresh balance
-        acct_1 = account_keys[1]
-        acct_2 = account_keys[2]
+        sender_account = account_keys[1]
+        receiver_account = account_keys[2]
         logger.info(
-            f"index_user_bank.py | Balance refresh accounts: {acct_1}, {acct_2}"
+            f"index_user_bank.py | Balance refresh accounts: {sender_account}, {receiver_account}"
+        )
+        user_id_accounts = refresh_user_balances(
+            session, redis, [sender_account, receiver_account]
+        )
+        index_tip(
```

Comment:
What if recipient 2 is an external wallet? Do we have any way to bail out of indexing that?